src/foam/mlang/mlang.js

```diff
@@ -2757,6 +2757,12 @@ foam.CLASS({
       name: 'exprs',
       documentation: 'The expressions to be evaluated and returned in the projection. Typically are Properties.',
     },
+    {
```

Comment:
so for example,
we class like FixedFee, but which also contains reference property value of which we need then we can gather all the data we need with one request with using Projection

Reply:
as alternative we could do 2 requests: 1. select (to gather all the objects) 2. projection to gather Reference property data